apps/web/src/ee/clerk/ClerkRoutes.tsx

```diff
@@ -0,0 +1,35 @@
+import { SignedIn, SignedOut } from '@clerk/clerk-react';
+import { Route } from 'react-router-dom';
+import { PublicPageLayout } from '../../components/layout/components/PublicPageLayout';
+import { ROUTES } from '../../constants/routes';
+import OrganizationListPage from './pages/OrganizationListPage';
+import QuestionnairePage from './pages/QuestionnairePage';
+import SignInPage from './pages/SignInPage';
+import SignUpPage from './pages/SignUpPage';
+
+export const ClerkRoutes = () => {
+  return (
+    <>
+      <Route
+        element={
+          <SignedOut>
+            <PublicPageLayout />
+          </SignedOut>
+        }
+      >
+        <Route path={ROUTES.AUTH_SIGNUP + '/*'} element={<SignUpPage />} />
```

Comment:
Let's replace these with /sign-in, and /sign-up

Reply:
I will replace the paths in a separate PR cause there are a lot of hardcoded references for this even outside of web, just to name few:
- various env vars in API (VERCEL_REDICERT_URI, CLIENT_SUCCESS_AUTH_REDIRECT...)
- tests in api
- places in CLI
- ...